src/WebJobs.Extensions.DurableTask/FunctionType.cs

```diff
@@ -1,12 +1,12 @@
 ﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the MIT License. See License.txt in the project root for license information.
 
-namespace Microsoft.Azure.WebJobs
+namespace Microsoft.Azure.WebJobs.Extensions.DurableTask
 {
     /// <summary>
     /// The type of a function.
     /// </summary>
-    internal enum FunctionType
+    public enum FunctionType
```

Comment:
Why was this made public? Is it just for unit testing? If so, the test project should already be configured to access internal types, so you shouldn't need to make this public.